src/waveresponse/_core.py

```diff
@@ -141,6 +142,51 @@ def _sort(dirs, vals):
     return dirs[sorted_args], vals[:, sorted_args]
 
 
+class _GridInterpolate:
```

Comment:
Rename to GridInterpolator to be more like scipy.interpolate nameing convention?